lib/fennec/stun.ex

```diff
@@ -17,15 +18,23 @@ defmodule Fennec.STUN do
     {:ok, {binary, %TURN{}}} | {:ok, :void} | {:error, reason :: term}
   def process_message(data, client, server, turn_state) do
     with secret        =  Auth.get_secret(),
-         {:ok, params} <- Jerboa.Format.decode(data, [secret: secret]),
+         {:ok, %Params{} = params} <- Jerboa.Format.decode(data, [secret: secret]),
```

Comment:
Why do we need to handle `ChannelData` as a separate case? Couldn't we just dispatch it to `Evaluator.ChannelData.service` in `Evaluator.service`? If `Evaluator.ChannelData.service` would return `{:void, new_turn_state}` the rest of the code works just fine.

Reply:
That would require adding a couple more clauses to `Evalutor` module, since it tries to authenticate the incoming STUN packet. Handling channel data here requires less changes. Also I think that it's a better place to show that `ChannelData` is something which is treated differently than regular STUN message.